### ISSUE TITLE
ANCOM documentation confusion

### ISSUE THREAD

**Issue by wasade** [biocore/scikit-bio#1344]
The `notes` section for `ancom` describes statistical tests that the authors recommend, but I believe there is a typo. The specific line is [here](https://github.com/biocore/scikit-bio/blob/master/skbio/stats/composition.py#L724), where it indicates that the authors recommend method `w` and `x` when "the number of groups is greater than 2", and in the same sentence, it then recommends methods `y` and `z` when "the number of groups is greater than 2." My guess is that the first example is supposed to be "less than". 

cc @mortonjt

**Comment by mortonjt** [comment]
Not quite - you can only run the `ttest_ind` and `mann-whitney` if there are two groups.

Fixing

**Comment by mortonjt** [comment]
Actually, I'm looking over the wording, and am a little confused where the confusion is stemming from. 

The first two methods are parametric, and the second two methods are non-parametric.  Do you have any suggestions on improving this?

**Comment by wasade** [comment]
Remove the repetition of "greater than 2"?

**Comment by jairideout** [comment]
The current text is correct but hard to parse. I followed up on #1346 with a suggested rewording.

**Comment by mortonjt** [comment]
Thanks!

**Comment by wasade** [comment]
Thanks!

**Comment by jairideout** [comment]
Fixed in #1346.
